config/templates/helm-labels.yaml.envsubst

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: builtin
+kind: LabelTransformer
+metadata:
+  name: labelTransformer
+labels:
+  app.kubernetes.io/instance: '{{ .Release.Name }}'
+  app.kubernetes.io/name: '{{ template "kyverno.name" . }}'
+  app.kubernetes.io/part-of: '{{ template "kyverno.name" . }}'
+  app.kubernetes.io/version: $VERSION
+fieldSpecs:
+- path: metadata/labels
+  create: true
+- kind: Deployment
+  path: spec/template/metadata/labels
+  create: true
```

Comment:
This should not be needed for helm.
I wonder if the sed magic is not safer. WDYT ?